taxi/tests/test-models.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+# Create your tests here.
+class ModelsTest(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test_name", country="test_country")
+        self.assertEqual(str(manufacturer), "test_name test_country")
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="test",
+            password="test123",
+            first_name="test_first1",
+            last_name="test_last1",
+        )
+        self.assertEqual(
+            str(driver),
+            f"{driver.username} ({driver.first_name} {driver.last_name})"
+        )
+
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test_name", country="test_country")
+        car = Car.objects.create(
+            model="test_model",
+            manufacturer=manufacturer,
+        )
+        self.assertEqual(str(car), "test_model")
+
+    def test_create_driver_with_license_number(self):
+        username = "test"
+        password = "test123"
+        license_number = "ABC12345"
+        driver = get_user_model().objects.create_user(
+            username=username,
+            password=password,
+            license_number=license_number,

```

Comment:
Ensure that the `Driver` model has a `license_number` field. If `get_user_model()` returns a model that does not include this field, the test will fail. Verify the model definition to confirm the presence of `license_number`.